scipy/optimize/_trustregion_exact.py

```diff
@@ -0,0 +1,413 @@
+"""Nearly exact trust-region optimization subproblem."""
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from scipy.linalg import (norm, get_lapack_funcs, solve_triangular,
+                          cho_solve)
+from ._trustregion import (_minimize_trust_region, BaseQuadraticSubproblem)
+
+__all__ = ['estimate_smallest_singular_value',
```

Comment:
I agree that this is a little weird, it is, however, consistent with the remaining of the `optimize` package: The other minimization methods all have similar names, for instance: `_minimize_neldermead`, `_minimize_trust_ncg`,  `_minimize_tnc`, `_minimize_dogleg`...